web/packages/teleport/src/types.ts

```diff
@@ -87,6 +87,9 @@ export interface TeleportFeatureRoute {
   path: string;
   exact?: boolean;
   component: React.FunctionComponent;
+  // useOwnLayout is used to ignore the ContentMinWidth that wraps
+  // every component. default = false
+  useOwnLayout?: boolean;
```

Comment:
nit: maybe rename this to `hasOwnLayout` or something since `useXXX` naming typically refers to hooks

Reply:
good call